src/Protocol/Protocol18x.cpp

```diff
@@ -1123,6 +1147,68 @@ void cProtocol180::SendDisplayObjective(const AString & a_Objective, cScoreboard
 
 
 
+void cProtocol180::SendSetSubTitle(const cCompositeChat & a_SubTitle)
+{
+	ASSERT(m_State == 3);  // In game mode?
+
+	cWorld * World = m_Client->GetPlayer()->GetWorld();
+	bool ShouldUseChatPrefixes = (World == nullptr) ? false : World->ShouldUseChatPrefixes();
+
+	cPacketizer Pkt(*this, 0x45);  // Title packet
+	Pkt.WriteVarInt32(1);  // Set action as, set
+
+	Pkt.WriteString(a_SubTitle.CreateJsonString(ShouldUseChatPrefixes));
+}
+
+
+
+
+
+void cProtocol180::SendSetSubTitle(const AString & a_SubTitle)
+{
+	ASSERT(m_State == 3);  // In game mode?
+
+	cPacketizer Pkt(*this, 0x45);  // Title packet
+	Pkt.WriteVarInt32(1);  // Set action as, set subtitle
+
+	Pkt.WriteString(Printf("{\"text\":\"%s\"}", EscapeString(a_SubTitle).c_str()));
+}
```

Comment:
We might want to rename this to `SendSetRawSubtitle()` and send the input string as-is, so that the `/title` command can send the entire string through here unparsed unmodified.